cardano-api/src/Cardano/Api/ProtocolParameters.hs

```diff
@@ -1693,6 +1717,7 @@ fromAlonzoPParams
     , protocolParamMaxValueSize        = Just _maxValSize
     , protocolParamCollateralPercent   = Just _collateralPercentage
     , protocolParamMaxCollateralInputs = Just _maxCollateralInputs
+    , protocolParamUTxOCostPerByte     = Nothing
```

Comment:
Adding a comment here explaining this is deprecated in Babbage would be useful

Reply:
Added comment.